### DISCUSSION TITLE
More noob questions… /mnt not mounting

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [kth5/archpower#53] [General]
Hey there!

Yet another noob question during install, and I've tried googlfu which has failed. 

I thought I was following the wiki steps accurately, but then encounter the message shown in the photo. 

Anyone have any "pro tips"? 

(I know this is such a basic question, but really eager to get passed the unfamiliar territory into enjoying my PowerBook again!)

 ![image](https://user-images.githubusercontent.com/105223045/204268842-93d81782-2d8c-460e-807c-95cf38b0a7a9.jpeg)

**Comment by kth5** [comment]
Is your future root-fs mounted to `/mnt`? It looks as if you only mounted `/mnt/boot/grub` for the bootstrap partition.

So yes, create your rootfs partition and format it, then mount it to `/mnt` first. Then you create `/mnt/boot/grub` and mount your bootstrap partition there.
